src/styles/global.ts

```diff
@@ -0,0 +1,22 @@
+import "sanitize.css";
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyle = createGlobalStyle`
+*{
+    margin : 0;
+    padding : 0;
+    box-sizing: border-box;
+}
+body{
+    margin : 0;
+    padding : 0;
+}
+a{
+    text-decoratiom: none;
+}
+ul, li{
+    list-style: none;
+}
+
+
+`;
```

Comment:
들여쓰기 및 태그(괄호) 띄어쓰기 하면 좋을 것 같아용!!! 😊
```suggestion
import { createGlobalStyle } from "styled-components"

export const GlobalStyle = createGlobalStyle`
  * {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
  }

  body {
    margin: 0;
    padding: 0;
  }

  a {
    text-decoration: none;
  }

  ul, li {
    list-style: none;
  }
`;
```